src/pages/Admin/Admin.styles.ts

```diff
@@ -1,24 +1,20 @@
 import styled from "styled-components";
 import { Wrapper as ToggleSwitch } from "components/atoms/ToggleSwitch/ToggleSwitch.styles";
 
-export const Wrapper = styled.div`
+interface WrapperProps {
+  hasSelectedVenue?: boolean;
+}
+
+export const Wrapper = styled.div<WrapperProps>`
   display: grid;
   height: 100vh;
-  grid-template-columns: 240px auto;
+  grid-template-columns: ${({ hasSelectedVenue }) =>
+    hasSelectedVenue ? "240px auto" : "auto"};
 
   position: relative;
   overflow-y: scroll;
-
   scrollbar-width: thin;
 
-  .no-venue-selected {
-    display: flex;
-    justify-self: center;
-    align-self: center;
-
-    font-size: 3rem;
-  }
```

Comment:
The deleted styles here aren't fully represented in the styles that had `hasSelectedVenue` added to them. Just want to confirm that this is intended/as expected.

Reply:
Yup, they were unused

Reply:
Cool :)